bin/pytest-litf

```diff
@@ -1,25 +1,41 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Pytest-LITF launcher
+"""
+
+from __future__ import print_function, absolute_import, division
```

Comment:
So, these came from my script creation snippet -- I know they're not really necessary, but since we're aiming for Python 2/3 cross-compatibility I thought it wouldn't hurt.

I can remove if you think it should be out.

Reply:
It looks good, thanks